src/material-experimental/mdc-slider/_slider-theme.scss

```diff
@@ -1,22 +1,43 @@
-// TODO: disabled until we implement the new MDC slider.
-// @import '@material/slider/mixins.import';
+@import '@material/slider/slider';
```

Comment:
`@use '@material/slider' as mdc-slider;` (otherwise use the `.import` file, doing it this way imports unprefixed names like `without-ripple`)